colcon_cmake/task/cmake/build.py

```diff
@@ -236,10 +237,10 @@ def _get_last_cmake_args_path(self, build_base):
                 job_args = self._get_make_arguments()
                 if job_args:
                     cmd += ['--'] + job_args
-            rc = await check_call(
+            completed = await check_call(
                 self.context, cmd, cwd=args.build_base, env=env)
-            if rc and rc.returncode:
-                return rc
+            if completed.returncode:
+                return completed.returncode
```

Comment:
This is also changing the return type.

Reply:
This change goes together with your other comment. The `_build` method now returns just the integer return code (instead of a `CompletedProcess` before).